packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.test.js

```diff
@@ -305,4 +305,39 @@ describe('useAutocomplete', () => {
       expect(input.value).to.equal('free');
     });
   });
+
+  it('should allow single value tuples or arrays', () => {
```

Comment:
Nit: this will make it more clear what the test is about at first glance.

```suggestion
  it('should allow tuples or arrays as value when multiple=false', () => {
```